src/components/About.astro

```diff
@@ -0,0 +1,54 @@
+<section class="grid">
+  <div class="about-information">
+    <h2>O firmie</h2>
+    <p class="large-body">
+      Regeneracją siłowników zajmujemy się od wielu lat. Doświadczenie
+      zdobywaliśmy poprzez wieloletnie zaangażowanie w&nbspremonty różnego
+      rodzaju siłowników hydraulicznych. Od 2018 roku jako jedyna firma w Polsce
+      wykonujemy regenerację siłowników do otwierania/zamykania dachu
+      w&nbspkabrioletach.
+    </p>
+    <div class="small-body-decoration">
+      <p class="small-body">
+        Regenerujemy siłowniki takich samochodów Audi, BMW, Mrecedes-Benz, Saab,
+        Porsche i&nbspwielu innych.
+      </p>
+    </div>
+
+    <div class="small-body-decoration">
+      <p class="small-body">Na naprawę udzielamy 12 miesięcy gwarancji.</p>
+    </div>
+  </div>
+  <div>
+    <img class="img-about-container" src="img/about.jpg" alt="" />
```

Comment:
all images might be replaced with Astro's [Image](https://docs.astro.build/en/guides/assets/)  for better performance. 

It's experimental for now, but might be worth a try (in another task)